pkgs/development/libraries/tbb/2020_3.nix

```diff
@@ -43,7 +43,7 @@ stdenv.mkDerivation rec {
     (fetchurl {
       name = "aarch64-darwin.patch";
       url = "https://github.com/oneapi-src/oneTBB/pull/258/commits/86f6dcdc17a8f5ef2382faaef860cfa5243984fe.patch";
-      sha256 = "sha256-JXqrFPCb3q1vfxk752tQu7HhApCB4YH2LoVnGRwmspk=";
+      hash = "sha256-ZAWzx/NOenTDBWt58sm9qEG4zC++92Xs2eSjxhGwZ4U=";
     })
```

Comment:
This should really be
```
    (fetchpatch {
      name = "aarch64-darwin.patch";
      url = "https://github.com/oneapi-src/oneTBB/pull/258/commits/86f6dcdc17a8f5ef2382faaef860cfa5243984fe.patch";
      hash = "sha256-+sNU8yEsVVmQYOCKmlNiyJfKmB/U0GKAmrydwkfrDFQ=";
    })
```
Using `fetchpatch` instead of `fetchurl` [avoids this type of problem entirely](https://nixos.org/manual/nixpkgs/unstable/index.html#fetchpatch).

Reply:
Good point, I didn't even catch that it's not fetchpatch :(